### ISSUE TITLE
Adding to shopping card pop up

### ISSUE THREAD

**Issue by jhhuynh42** [eezcommerce/eezcommerce#230]
click on the add to card cayses product detail to pop up for a second

**Comment by lucacataldo** [comment]
Nice catch 👍

**Comment by lucacataldo** [comment]
whoever tackles this: use the jquery ":not()" selector to not trigger the modal when the button is clicked, but anything else in the product card is

**Comment by tselouie** [comment]
yeah i was gonna fix this along when i figure out how to add shoppingcart to a variable in the store.

it will be similar to the deletebutton in products
